website/content/docs/plugins/index.mdx

```diff
@@ -1,21 +1,46 @@
 ---
 description: |
-  Packer plugins let you add Packer functionality without modifying
-  the core source code. Packer plugins are able to add new builders,
-  provisioners, hooks, and more.
+  This topic provides overview information about installing and loading Packer plugins
 page_title: Plugins
 ---
 
-# Packer Plugins
+# Plugin installation overview
 
-Packer plugins are separate, standalone applications that perform tasks during each build.
+This topic provides overview information about installing and loading Packer plugins. Plugins are standalone applications that perform additional tasks during each build.
 
-During a `packer build`, the process list shows `packer-` prefixed applications. One of those applications is the Packer binary, and the rest are plugins. Packer launches one plugin process for each component in the build.
+## Introduction
 
-The Packer binary has a set of built-in plugins that it can find and run automatically. You can also define a list of external plugins in your template for Packer to run and communicate with throughout the build. These external plugins extend Packer functionality without modifying the core source code.
+The Packer binary includes a set of built-in plugins that run automatically, but you can also define a list of external plugins in your template for Packer to run and communicate with throughout the build. These external plugins extend Packer functionality without modifying the core source code. Packer launches one plugin process for each component in the build.
```

Comment:
Just to avoid confusion here, we don't embed plugins with Packer anymore, but there are components that are servable by Packer

```suggestion
The Packer binary includes a set of built-in components that are automatically usable. You can also install external plugins for Packer to run and communicate with throughout the build. These external plugins extend Packer functionality without modifying the core source code. Packer launches one plugin process for each component in the build.
```